ethereum/contracts/SuperAdjudicator.sol

```diff
@@ -0,0 +1,147 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.4;
+
+import {IArbitrator} from '@kleros/erc-792/contracts/IArbitrator.sol';
+
+uint256 constant STARKNET_PRIME = 2**251 + 17 * 2**192 + 1;
+
+// XXX: this selector computation probably isn't correct; verify it!
+uint256 constant MASK_250 = 2**250 - 1;
+uint256 constant ZORRO_SUPER_ADJUDICATE_SELECTOR = uint256(
```

Comment:
Ouch, you'd think they'd expose a `computeSelector` method on their own contract to avoid having to duplicate this logic.